### PR TITLE
[Linux] Added SetVersion.sh

### PR DESCRIPTION
**Pull request by SlavaRa**

### CODE REVIEW

File: FlashDevelop/FlashDevelop.csproj [hunk 1]
```diff
@@ -257,13 +257,20 @@
...
+  <PropertyGroup Condition=" '$(OS)' == 'Unix' ">
+    <PreBuildEvent>"$(SolutionDir)\SetVersion.sh" "$(SolutionDir)"</PreBuildEvent>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(OS)' != 'Unix' And '$(Platform)' == 'AnyCPU' ">
     <PreBuildEvent>"$(SolutionDir)\SetVersion.bat" "$(SolutionDir)" &amp;&amp; ^
 IF EXIST "$(TargetPath)" move /Y "$(TargetPath)" "$(TargetDir)$(TargetName)86$(TargetExt)"</PreBuildEvent>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(OS)' == 'Unix' And '$(Platform)' == 'AnyCPU' ">
+    <PreBuildEvent>"$(SolutionDir)\SetVersion.sh" "$(SolutionDir)" &amp;&amp; ^
+IF EXIST "$(TargetPath)" move /Y "$(TargetPath)" "$(TargetDir)$(TargetName)86$(TargetExt)"</PreBuildEvent>
```
Comment (chipshort) [inline FlashDevelop/FlashDevelop.csproj]: Is this supposed to run on a unix machine? If so, what about the move and IF EXIST. Those are windows specific. Or does mono do some magic here?

File: SetVersion.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+target_dir=$1
+cd $target_dir
+mono External/Tools/SetVersion/SetVersion.exe FlashDevelop/Properties/AssemblyInfo.cs "%APPVEYOR_BUILD_NUMBER%"
```
Comment (chipshort) [inline SetVersion.sh]: I'm not sure about the `%APPVEYOR_BUILD_NUMBER%` wouldn't it be "$APPVEYOR_BUILD_NUMBER" on linux / unix?

File: FlashDevelop/FlashDevelop.csproj (hunk 1 above)
Comment (SlavaRa) [inline FlashDevelop/FlashDevelop.csproj]: all ok https://travis-ci.org/SlavaRa/flashdevelop

File: SetVersion.sh (hunk 2 above)
Comment (SlavaRa) [inline SetVersion.sh]: all ok https://travis-ci.org/SlavaRa/flashdevelop

File: FlashDevelop/FlashDevelop.csproj (hunk 1 above)
Comment (Neverbirth) [inline FlashDevelop/FlashDevelop.csproj]: You sure you end up with both FlashDevelop and FlashDevelop64?

Comment (SlavaRa) [inline FlashDevelop/FlashDevelop.csproj]: At the moment I have a build of 64 bit version disabled

Comment (Neverbirth) [inline FlashDevelop/FlashDevelop.csproj]: Heh, so you are not running that IF EXIST and move then.

Comment (SlavaRa) [inline FlashDevelop/FlashDevelop.csproj]: Travis-CI config https://github.com/SlavaRa/flashdevelop/blob/development-linux/.travis.yml
I can add a build there for a 64-bit system, but this has nothing to do with the configuration of the project, it is made in analogy with the Windows version
I do not understand, what is the problem now?

Comment (Neverbirth) [inline FlashDevelop/FlashDevelop.csproj]: It seems the solution wouldn't successfully compile when involving both x86 and AnyCPU builds as that command is specific to Windows. Could you check? Even if we don't have both builds to at least to not have the AnyCPU one to fail if it's the case right now.

Comment (SlavaRa) [inline FlashDevelop/FlashDevelop.csproj]: AnyCPU is used for PluginCore https://github.com/SlavaRa/flashdevelop/blob/development-linux/.travis.yml#L15
But in order to use it for the editor when build on Linux there will be some necessary modifications
Of course I can remove the lines 270-273 as unused

Comment (SlavaRa) [inline FlashDevelop/FlashDevelop.csproj]: @Neverbirth ?

Comment (Neverbirth) [inline FlashDevelop/FlashDevelop.csproj]: Which modifications are needed for AnyCPU?

Comment (SlavaRa) [inline FlashDevelop/FlashDevelop.csproj]: If I was sure about what to do, I would have done it already)
This PR allows my team to run the assembly now without any unnecessary actions. I think that the guys in the process of work will adjust the build for AnyCPU

Comment (Neverbirth) [inline FlashDevelop/FlashDevelop.csproj]: I see, I'm sure one of the changes is that IF EXIST and move part heh.

**Pull request merged**